server/cpho/views/indicator_metadata.py

```diff
@@ -109,6 +107,54 @@ class Meta:
         ),
     )
 
+    def clean(self):
+        cleaned_data = super().clean()
+        return cleaned_data
```

Comment:
I don't think we need this function because all we are doing is calling the super.clean() function.